tests/test_thunderbird.py

```diff
@@ -349,10 +349,17 @@ def send_email(tb, sign=False, encrypt=False, inline=False, attachment=None):
                 button('Protect subject').doActionNamed('press')
     except tree.SearchError:
         pass
+    # if there is a dialog window, then there must have been some issue sending
+    try:
+        tb.app.dialog('.*')
```

Comment:
If going this way, maybe log the dialog title, or even the whole content? Or will it be included in the output already?

Reply:
Good point. Just added that. Makes the code slightly more complex because of nested exceptions but I guess that's ok.

Reply:
This proved to be too error-prone.